e2e-tests/README.md

```diff
@@ -2,15 +2,20 @@
 
 This is an automation tests folder for Virtual Visits sample, which contains all UI tests.
 
-## Build and Run End to End tests
+## Build and Run End to End tests locally
 
-- To install supported browsers, use the following command:
+- Before running the tests make sure that the environment variable for connection string is set. [Learn more about environment variables and how to set them.](https://github.com/Azure-Samples/communication-services-virtual-visits-js#environment-variables)
+
+- To install dependencies, use the following command:
 
 ```
+npm install
 npx playwright install
```

Comment:
I followed the steps in under `e2e-tests` directory, it didn't go as the instruction unfortunately.. 

![image](https://user-images.githubusercontent.com/94404633/175439451-f150fb3a-c671-41e3-813d-12a31371a12e.png)

It worked after run the following command as the screenshot above suggested.

`npm install -D @playwright/test`

![image](https://user-images.githubusercontent.com/94404633/175439596-52653bf3-c1df-4af6-8478-3599f4230bc9.png)